imap_processing/cli.py

```diff
@@ -215,13 +219,13 @@ class ProcessInstrument(ABC):
             'descriptor': 'sci',
             'version': 'v00-01',
             'start_date': '20231212'
-        }]"
+        }]" .
     start_date : str
-        The start date for the output data. Format: YYYYMMDD
+        The start date for the output data. Format: YYYYMMDD .
```

Comment:
The extra space before the period on these looks out of place. Could we reword the sentence to include the format instead? (similar below if you take this suggestion)
```suggestion
        The start date for the output data in **YYYYMMDD** format.
```